### ISSUE TITLE
Loss function research

### ISSUE THREAD

**Issue by OnurSefa** [OnurSefa/OSOSAART#5]
Encoder requires loss function. At this moment we implemented cross entropy loss but we should be sure about the effectiveness of this loss function or we should use another loss functions for our purpose.

**Comment by OnurSefa** [comment]
Cross Entropy Loss function works fine currently because we changed our encoder to a classification model.
